src/libs/actions/IOU.ts

```diff
@@ -6339,6 +6339,87 @@ function approveMoneyRequest(expenseReport: OnyxTypes.Report | EmptyObject, full
     API.write(WRITE_COMMANDS.APPROVE_MONEY_REQUEST, parameters, {optimisticData, successData, failureData});
 }
 
+function unapproveMoneyRequest(expenseReport: OnyxTypes.Report | EmptyObject) {
```

Comment:
Lets try to use this naming convention now - I bet you make this function when I still had the old name in the doc, sorry! :D

```suggestion
function unapproveExpenseReport(expenseReport: OnyxTypes.Report | EmptyObject) {
```

Reply:
no worriess